src/core/utils/useReaderPlayer.tsx

```diff
@@ -1,51 +1,81 @@
-import { useGetV1UserLoans } from "../publizon/publizon";
 import { Manifestation } from "./types/entities";
 import { getManifestationIsbn } from "../../apps/material/helper";
 import {
   getOrderIdByIdentifier,
-  getReaderPlayerType
+  getReaderPlayerType,
+  getReservedReservation
 } from "../../components/reader-player/helper";
-import { mapPublizonLoanToLoanType } from "./helpers/list-mapper";
 import { isAnonymous } from "./helpers/user";
+import useOnlineAvailabilityData from "../../components/availability-label/useOnlineAvailabilityData";
+import { hasCorrectAccess } from "../../components/material/material-buttons/helper";
+import {
+  useGetV1UserLoans,
+  useGetV1UserReservations
+} from "../publizon/publizon";
+import {
+  mapPublizonLoanToLoanType,
+  mapPublizonReservationToReservationType
+} from "./helpers/list-mapper";
 
 const useReaderPlayer = (manifestations: Manifestation[] | null) => {
   const isUserAnonymous = isAnonymous();
-  const { data } = useGetV1UserLoans(
+  const hasManifestations = !!manifestations?.length;
+
+  const type =
+    hasManifestations && hasCorrectAccess("Ereol", manifestations)
+      ? getReaderPlayerType(manifestations)
+      : null;
+  const identifier = hasManifestations
+    ? getManifestationIsbn(manifestations[0])
+    : null;
```

Comment:
Again I wonder about the scope/intent of this hook. It accepts an array of manifestations and then only seems to work with one of them. I guess it is based on the intention that these manifestations are somehow related.

I wonder whether it would be preferable to only let the hook accept a single manifestation (or even ISBN) and then let the surrounding code determine which isbn to pass. This might also make the code more understandable.

0-based array access is better here with `hasManifestations` but I am not a fan.